test/integration/narrow/routes/incident/details-of-the-smell.test.js

```diff
@@ -35,8 +39,51 @@ describe('Experience test', () => {
     expect(response.statusCode).toBe(200)
   })
 
+  test('POST /details-of-the-smell route returns 302', async () => {
+    mockSession = {
+      firstName: 'Mr',
+      lastName: 'Test',
+      addressLine1: 'Institute of test',
+      addressLine2: '',
+      townOrCity: 'Teston',
+      county: 'Testshire',
+      postcode: 'T3 5TR',
+      email: 'tester@test.com',
+      phonenumber: '1471',
+      strength: 'strong smell that may make your hair or clothes smell',
+      atHome: 'No',
+      location: 'elsewhere',
+      description: 'bad',
+      date: '2021-08-25T00:00:00.000Z',
+      time: '14:35'
+    }
+
+    const options = {
+      method: 'POST',
+      url: '/details-of-the-smell',
+      payload: {
+        description: 'test description',
+        date: '2017-08-25',
+        time: '14:35'
+      }
+    }
+
+    const response = await server.inject(options)
+    expect(response.statusCode).toBe(302)
+  })
+
   afterEach(async () => {
     jest.clearAllMocks()
     await server.stop()
   })
 })
+
```

Comment:
Looks like these commented out lines are left over from the confirmation test, and can be removed.

Reply:
Thanks!